src/pyze/cli/status.py

```diff
@@ -64,6 +64,7 @@ def run(parsed_args):
         ['AC state', hvac['hvacStatus'] if 'hvacStatus' in hvac else None],
         ['AC start at', hvac_start] if hvac_start else None,
         ['External temperature', "{}°C".format(hvac['externalTemperature'])],
+        ['Battery temperature', "{}°C".format(status['batteryTemperature'])] if status['batteryTemperature'] else None,
```

Comment:
This needs to be phrased as the following, as with the other similar lines, else it's pointless as a guard:
```suggestion
        ['Battery temperature', "{}°C".format(status['batteryTemperature'])] if 'batteryTemperature' in status else None,
```